communication/controller/src/main/java/org/openbase/jul/communication/controller/AbstractControllerServer.java

```diff
@@ -1238,10 +1247,9 @@ public boolean isShutdownInProgress() {
      */
     @Override
     public String toString() {
-        try {
-            return getClass().getSimpleName() + "[" + informer.getScope().toString() + "]";
-        } catch (NotAvailableException ex) {
-            return getClass().getSimpleName() + "[]";
+        if (publisher == null) {
+            return getClass().getSimpleName();
         }
+        return getClass().getSimpleName() + "[" + publisher.getScope() + "]";
```

Comment:
I think `publisher.getScope()` now returns our scope proto type whose toString() prints a multi line string. Maybe we this should use the `ScopeProcesser.generateStringRep()` method.